### DISCUSSION TITLE
Rendering at high resolution

### DISCUSSION THREAD

**Discussion by persius77** [lllyasviel/Fooocus#2879] [Q&A] [answered]
I'm still rather new to Fooocus but progress is being made.    I have been playing with wilderness scenes and would like to ask for advice on how to render out higher resolution detailed scenes.  

I've included an image that I would like to render and print poster size (16 x 20 inch/ 4800 x 6000).  What render settings should start with to ensure sharpest detail?  The image attached was rendered at 768 x 1280. 

I've watched a few  youtube videos explaining how to ad customized resolution settings.  I created a new AR setting (3000 x 2400) thinking that would work but the renders were.... mutated.  Nothing even close to what I wanted.   Clearly, something went wrong but I have no idea what or where. 

Any advice will be appreciated.
Thank you.

![2024-04-03_01-08-04_5124](https://github.com/lllyasviel/Fooocus/assets/165508031/773aa6b9-4503-4f20-9b62-4fea88ebb6f4)

**Comment by mashb1t** [comment, marked as answer]
Hey @persius77, you'd have to generate an image in a standard resolution (fooocus defaults) and then upscale multiple times. Fooocus allows upscaling until 4x afaik (2x 2x) as an iterative process.
Please find more information about why higher resolutions aren't supported in https://github.com/lllyasviel/Fooocus/pull/1617.

**Reply by wisdombox44** [reply]
You may try upscale one of the best free upscale "upscaly" visit upscaly.org It works differently then Fooocus upscale.   it does not add extra detail or change the image. Basically protects the detail and enhances. Which has multiple different upscale models. The Result will be very similar to Fooocus "2X upscale fast"
I still use time to time. Totally Free

**Comment by persius77** [comment]
Hi Manuel.
Thank you for replying.  I rendered out an image at 1536 x 640.  I took
your advice by upscaling it with 2x and then taking that image and
upscaling it again at 2x.  That seems to have worked.     This may seem
like a basic question to you but if I need to adjust the image but adding a
"top" to it.  Will Fooocus understand all the requirements to add the same
level of detail to the upscaled image, or, will it just add the basics and
leave it lower resolution?   Sometimes just looking at the image doesn't
really provide the answer.

Thanks again.

**Reply by mashb1t** [reply]
Fooocus can work with higher resolutions, but please consider that it might take more time and might also be repeating pattern as the AI has not been trained on high resolutions.

**Comment by persius77** [comment]
Hi Wisdombox.

I found "upscaly" and installed it.   You obviously know a little more
about upscaling than I do at this time.  I did read about another upscaler
called "Topaz Gigapixel".   From your experience, which one would YOU
recommend relying on?   I don't know enough to understand if there is a
difference or if one is superior to the other based on what I am trying to
do with my work.
Any thoughts?


Thanks.

**Comment by wisdombox44** [comment]
Topaz Gigapixel is paid software that operates with its own upscale models. Upscaly, on the other hand, is totally free and comes with various upscale models, with more than enough provided by default. If you need additional models, you can download them from https://openmodeldb.info/. Both software options perform similar jobs.

Paid vs. Free: They lack creative upscale features like Fooocus. If you want your images upscaled closely to the original without any alteration, use them. Fooocus upscale is also powerful and offers creative upscale options. Every image is a different scenario, so the best approach is to experiment with them for your specific needs. As mentioned before, Fooocus "Upscale (Fast 2x)" functions similarly to Topaz Gigapixel and Upscaly. For some images, I use Fooocus "Upscale (2x)" with the "lightning" preset, which surprisingly yields good results for some images but not for every image.
